### ISSUE TITLE
Issue with docker-compose on 'main' branch

### ISSUE THREAD

**Issue by jeremyf** [samvera/hyrax#4896]
tl;dr: for SHA ab6c75616 `docker-compose up` works, for SHA efee62e00bbeee6121320ebd45ab39f4b5950a8d `docker-compose` fails.  

When working on Hyrax without a fix for this you may need to do one of two things:

1) locally revert 38aed2d0b.
2) checkout 38aed2d0b and work from that point.

## Git Log Summary between two commits

* efee62e00 — Removing unused method Jeremy Friesen, (2021-04-29) (HEAD -> main, origin/main)
* 1940df7cf — Merge pull request #4893 from samvera/light-entrypoint Jeremy Friesen, (2021-04-29)
* 38aed2d0b — build docker images with ghcr.io names in compose tamsin johnson, (2021-04-28) (origin/light-entrypoint)
* 9aca17759 — remove chowns from image entrypoint tamsin johnson, (2021-04-28)

## Error messages for efee62e00bbeee6121320ebd45ab39f4b5950a8d

These are the errors I get:

```sh
$ docker-compose up
Docker Compose is now in the Docker CLI, try `docker compose up`

Pulling db_migrate (ghcr.io/samvera/dassie:)...
ERROR: name unknown
```

```sh
❯ docker compose up
[+] Building 0.5s (13/37)
 => [db_migrate internal] load build definition from Dockerfile                          0.0s
 => => transferring dockerfile: 64B                                                      0.0s
 => [ghcr.io/samvera/dassie-worker internal] load build definition from Dockerfile       0.0s
 => => transferring dockerfile: 2.59kB                                                   0.0s
 => [fcrepo internal] load build definition from Dockerfile                              0.0s
 => => transferring dockerfile: 71B                                                      0.0s
 => [ghcr.io/samvera/dassie internal] load build definition from Dockerfile              0.0s
 => => transferring dockerfile: 32B                                                      0.0s
 => [db_migrate internal] load .dockerignore                                             0.0s
 => => transferring context: 34B                                                         0.0s
 => [ghcr.io/samvera/dassie-worker internal] load .dockerignore                          0.0s
 => => transferring context: 34B                                                         0.0s
 => [fcrepo internal] load .dockerignore                                                 0.0s
 => => transferring context: 34B                                                         0.0s
 => [ghcr.io/samvera/dassie internal] load .dockerignore                                 0.0s
 => => transferring context: 34B                                                         0.0s
 => ERROR [db_migrate internal] load metadata for ghcr.io/samvera/dassie:latest          0.2s
 => [ghcr.io/samvera/dassie internal] load metadata for docker.io/library/ruby:2.7.2-al  0.0s
 => CANCELED [ghcr.io/samvera/dassie-worker internal] load build context                 0.1s
 => => transferring context: 4.25MB                                                      0.1s
 => [ghcr.io/samvera/dassie-worker hyrax-base 1/7] FROM docker.io/library/ruby:2.7.2-al  0.0s
 => CANCELED [fcrepo internal] load metadata for ghcr.io/samvera/fcrepo4:4.7.5           0.1s
------
 > [db_migrate internal] load metadata for ghcr.io/samvera/dassie:latest:
------
The new 'docker compose' command is currently experimental. To provide feedback or request new features please open issues at https://github.com/docker/compose-cli
failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: ghcr.io/samvera/dassie:latest: not found
```

**Comment by jeremyf** [comment]
I ran `docker-compose down -v` then `docker-compose build`

Now I'm getting:

```
------
 > [hyrax-engine-dev 4/4] RUN RAILS_ENV=production SECRET_KEY_BASE='fakesecret1234' DB_ADAPTER=nulldb DATABASE_URL='postgresql://fake' bundle exec rake assets:precompile:
#15 0.524 fatal: not a git repository (or any of the parent directories): .git
#15 0.608 bundler: failed to load command: rake (/usr/local/bundle/bin/rake)
#15 0.608 Bundler::GemNotFound: Could not find devise-4.7.3 in any of the sources
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/spec_set.rb:86:in `block in materialize'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/spec_set.rb:80:in `map!'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/spec_set.rb:80:in `materialize'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/definition.rb:170:in `specs'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/definition.rb:237:in `specs_for'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/definition.rb:226:in `requested_specs'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/runtime.rb:101:in `block in definition_method'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/runtime.rb:20:in `setup'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler.rb:149:in `setup'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/setup.rb:20:in `block in <top (required)>'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/ui/shell.rb:136:in `with_level'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/ui/shell.rb:88:in `silence'
#15 0.608   /usr/local/lib/ruby/2.7.0/bundler/setup.rb:20:in `<top (required)>'
------
executor failed running [/bin/sh -c RAILS_ENV=production SECRET_KEY_BASE='fakesecret1234' DB_ADAPTER=nulldb DATABASE_URL='postgresql://fake' bundle exec rake assets:precompile]: exit code: 1
ERROR: Service 'app' failed to build
```

**Comment by no-reply** [comment]
see #4897. these are now the same issue.

**Comment by no-reply** [comment]
is this still an active issue?
